src/test/java/com/google/devtools/build/lib/rules/apple/DottedVersionTest.java

```diff
@@ -46,6 +46,7 @@ public void testCompareTo() throws Exception {
         .addEqualityGroup(DottedVersion.fromString("10.0.0.10A255"))
         .addEqualityGroup(DottedVersion.fromString("10.2"))
         .addEqualityGroup(DottedVersion.fromString("10.2.0.10P99q"))
+        .addEqualityGroup(DottedVersion.fromString("10.10.simulator.internal"), DottedVersion.fromString("10.10"))
```

Comment:
Would you want to support a build version still?

`10.10.simulator.internal.10A255`

Reply:
The descriptive segments will stay by the end of the version number.